libr/core/cmd_open.c

```diff
@@ -880,8 +880,8 @@ R_API void r_core_file_reopen_debug(RCore *core, const char *args) {
 		return;
 	}
 	int bits = core->assembler->bits;
-	char *oldname = r_file_abspath (binpath);
-	char *newfile = r_str_newf ("dbg://%s %s", oldname, args);
+	char *escaped_path = r_str_arg_escape (binpath);
+	char *newfile = r_str_newf ("dbg://%s %s", escaped_path, args);
```

Comment:
maybe we can use a different char than space to separate program name and arguments (as a hack)

Reply:
If we do this, we don't need to escape the file path.
We generate a URL which `radare2` can recognise here. As we don't specify special char to separate URL, it's better for us not to change it.